mteb/tasks/Audio/AudioClassification/eng/VoxPopuliGenderID.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import annotations
+
+from mteb.abstasks.Audio.AbsTaskAudioClassification import (
+    AbsTaskAudioClassification,
+)
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class VoxPopuliGenderID(AbsTaskAudioClassification):
+    metadata = TaskMetadata(
+        name="VoxPopuliGenderID",
+        description="Classification of speech samples by speaker gender (male/female) from European Parliament recordings.",
+        reference="https://huggingface.co/datasets/facebook/voxpopuli",
+        dataset={
+            "path": "facebook/voxpopuli",
+            "name": "en",  # This selects the english config/subset
+            "revision": "719aaef8225945c0d80b277de6c79aa42ab053d5",
+        },
+        type="AudioClassification",
+        category="a2t",
+        eval_splits=["validation", "test"],
```

Comment:
Let's only take the test split.
```suggestion
        eval_splits=["test"],
```